units/INA1001/INA1001_unit.bp

```diff
@@ -1,9 +1,11 @@
+-- T1 air scout
+
 UnitBlueprint {
     Abilities = {
         ArtillerySupport = {
             LockoutDuration = 1,
             MaxSimultaniousArtillerySupport = -1,
-            Range = 18,
```

Comment:
ooh you revel my sneaky change :D  I can move it in another PR, but not a real change at all.  on gunship its weapn range, but on scout it need be bigger while it dont stay on place.